deployment/ansible/templates/epoch.yaml.j2

```diff
@@ -1,11 +1,15 @@
 ---
 {% if 'epoch' in group_names %}
 peers:
-    {%for host in ansible_play_hosts | difference([inventory_hostname]) %}
+    {% for host in ansible_play_hosts | difference([inventory_hostname]) %}
 - "http://{{ hostvars[host]['ansible_default_ipv4']['address'] }}:3013/"
     {% endfor %}
 {%- endif %}
 
+keys:
+    dir: {{ config.keypair.dir }}
```

Comment:
I see this is "keys". I wonder if it shall be "./keys" or contain `{{ project_root }}`. Did you test locally?

Reply:
/me testing locally

Reply:
Yes, I tested it locally. Looks like it's relative to the "epoch" root

Reply:
Local test passed. With "keys".